build.gradle

```diff
@@ -43,18 +46,15 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-json'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
-	implementation "com.amazonaws:aws-java-sdk-sns:1.12.149"
+	implementation 'com.amazonaws:aws-java-sdk-sns:1.12.237'
```

Comment:
Atomic commits please for Dependencies - if we need to rollback this is more of a pain 😆